src/test/regress/output/multi_alter_table_statements.source

```diff
@@ -452,7 +452,79 @@ ORDER BY attnum;
 (29 rows)
 
 \c - - - :master_port
+-- check we don't intercept DDL commands if propagation is turned off
+SET citus.enable_ddl_propagation to false;
+-- table rename statement can be performed now
+ALTER TABLE lineitem_alter RENAME TO lineitem_renamed;
+-- verify rename is performed
+SELECT relname FROM pg_class WHERE relname = 'lineitem_alter' or relname = 'lineitem_renamed';
+     relname      
+------------------
+ lineitem_renamed
+(1 row)
+
+-- revert it to original name
+ALTER TABLE lineitem_renamed RENAME TO lineitem_alter;
+-- this column is added to master table and not workers
+ALTER TABLE lineitem_alter ADD COLUMN not_added_column int;
+-- verify newly added column is not present in a worker shard
+\c - - - :worker_1_port
+SELECT not_added_column FROM lineitem_alter_103000 LIMIT 0;
+ERROR:  column "not_added_column" does not exist
+LINE 1: SELECT not_added_column FROM lineitem_alter_103000 LIMIT 0;
+               ^
+\c - - - :master_port
+-- ddl propagation flag is reset to default, disable it again
+SHOW citus.enable_ddl_propagation;
+ citus.enable_ddl_propagation 
+------------------------------
+ on
+(1 row)
+
+SET citus.enable_ddl_propagation to false;
+-- following query succeeds since it accesses an previously existing column
+SELECT l_orderkey FROM lineitem_alter LIMIT 0;
+ l_orderkey 
+------------
+(0 rows)
+
+-- make master and workers have the same schema again
+ALTER TABLE lineitem_alter DROP COLUMN not_added_column;
+-- now this should succeed
+SELECT * FROM lineitem_alter LIMIT 0;
+ l_orderkey | l_partkey | l_suppkey | l_linenumber | l_quantity | l_extendedprice | l_discount | l_tax | l_returnflag | l_linestatus | l_shipdate | l_commitdate | l_receiptdate | l_shipinstruct | l_shipmode | l_comment | null_column 
+------------+-----------+-----------+--------------+------------+-----------------+------------+-------+--------------+--------------+------------+--------------+---------------+----------------+------------+-----------+-------------
+(0 rows)
+
+-- previously unsupported statements are accepted by postgresql now
+ALTER TABLE lineitem_alter ALTER COLUMN l_orderkey SET STATISTICS 100;
+ALTER TABLE lineitem_alter DROP CONSTRAINT IF EXISTS non_existent_contraint;
+NOTICE:  constraint "non_existent_contraint" of relation "lineitem_alter" does not exist, skipping
+ALTER TABLE lineitem_alter SET WITHOUT OIDS;
+-- even distribution column can be dropped however postgresql prevents this.
+ALTER TABLE lineitem_alter DROP COLUMN l_orderkey;
+ERROR:  cannot drop table lineitem_alter column l_orderkey because other objects depend on it
+DETAIL:  table lineitem_alter depends on table lineitem_alter column l_orderkey
+HINT:  Use DROP ... CASCADE to drop the dependent objects too.
+-- this is completely wrong, there can't be unique index on l_partkey
+-- Citus would have prevented that
```

Comment:
> We are removing all the validations on DDL statements with this work . User is expected to know what they are doing.

I mean, when I read the comment, I didn't get why we can't create a unique index on l_partkey in Citus. Then I noticed that this is a non-partition column.